src/test/java/uk/gov/pay/products/infra/DropwizardAppWithPostgresRule.java

```diff
@@ -102,12 +93,10 @@ public DatabaseTestHelper getDatabaseTestHelper() {
     }
 
     private void registerShutdownHook() {
-        Runtime.getRuntime().addShutdownHook(new Thread(() -> {
-            postgres.stop();
-        }));
+        Runtime.getRuntime().addShutdownHook(new Thread(postgres::stop));
     }
 
-    private JpaPersistModule createJpaModule(final PostgresDockerRule postgres) {
+    private void createJpaModule(final PostgresDockerRule postgres) {
```

Comment:
This looked a bit suspicious, it's used in a single place which does nothing with the response hence Intellij say's make it `void`. It doesn't look like the method has any lasting effect, it creates a `JpaPersistModule` but doesn't register it or anything. I commented out the single place its used and all the tests passed, so I think this method can be removed (to be safe I consulted Krishna and George who also think it looks unnecessary so probably safe..)